src/app/pages/view-secret-key/components/view-secret-key-message.tsx

```diff
@@ -0,0 +1,17 @@
+import { isFullPage } from '@app/common/utils';
+import { Body } from '@app/components/typography';
+
+interface ViewSecretKeyMessageProps {
+  wordCount: number;
+}
+export function ViewSecretKeyMessage(props: ViewSecretKeyMessageProps): JSX.Element {
+  const { wordCount } = props;
+
+  return (
+    <Body textAlign={isFullPage ? 'center' : 'left'}>
```

Comment:
We should use the array syntax here, something like

```suggestion
    <Body textAlign={['center', 'center', 'left']}>
```

Reply:
I don't know this syntax, so will look into it.

Reply:
It looks like we have breakpoints setup in the UI library, so I'll try to refactor to use them ...if going to hold up I'll just create an issue and do it next.